### PR TITLE
Tables support for IndexedVarArray

### PR DESCRIPTION
**Pull request by trulsf**
There will probably some alignment with JuMP if Tables support is added to JuMP (ref. https://github.com/jump-dev/JuMP.jl/issues/3096)

### CODE REVIEW

**Comment by trulsf** [comment]
The latest changes depend on the availability of Tables support in JuMP (see https://github.com/jump-dev/JuMP.jl/pull/3104) and should await merging until the PR is merged.

File: src/tables.jl
Comment (hellemo) [inline src/tables.jl:1]: Could this be `AbstractSparseArray`? I guess we are about to delete SparseVarArray in another PR soon.

**Review by hellemo — APPROVED** [review]
Looks good!

File: src/tables.jl
Comment (trulsf) [reply src/tables.jl:1]: The implementation of _rows assumes a `data` property exists which is not available in general for AbstractSparseArray. Agree that SparseVarArray just can be removed here.

**Comment by hellemo** [comment]
> The latest changes depend on the availability of Tables support in JuMP (see [jump-dev/JuMP.jl#3104](https://github.com/jump-dev/JuMP.jl/pull/3104)) and should await merging until the PR is merged.

How about just adding rowtable without extending JuMP function while we wait to get all the other changes in and merged and have tests in place, and rework to extend JuMP methods after https://github.com/jump-dev/JuMP.jl/pull/3104 is merged, would that work?

**Comment by trulsf** [comment]
I have moved rowtable out of JuMP.Containers and fixed tests to run with latest JuMP release. Maybe include this as part of 0.7 release and move back and fix tests when new JuMP version is available?

**Comment by hellemo** [comment]
> I have moved rowtable out of JuMP.Containers and fixed tests to run with latest JuMP release. Maybe include this as part of 0.7 release and move back and fix tests when new JuMP version is available?

Thanks! Good idea, I originally planned to wait for the JuMP release, but I think as long as we don't export `rowtables` and make it clear that we don't support tables, we should be good. If we want to make it super clear we could rename it to`_rowtables`, but I don't think that's necessary.

**Pull request merged**
